Makefile

```diff
@@ -206,7 +206,11 @@ $(VENDOR_DIR): glide.lock glide.yaml
 deps: $(VENDOR_DIR)
 
 app/controllers.go: $(DESIGNS) $(GOAGEN_BIN) $(VENDOR_DIR)
-	$(GOAGEN_BIN) bootstrap -d ${PACKAGE_NAME}/${DESIGN_DIR}
+	$(GOAGEN_BIN) app -d ${PACKAGE_NAME}/${DESIGN_DIR}
+	$(GOAGEN_BIN) controller -d ${PACKAGE_NAME}/${DESIGN_DIR} -o controllers/ --pkg controllers --app-pkg app
```

Comment:
Although the controllers directory will contain multiple controllers we should decide if we want to make this plural or singular. I mean, look around, the other folders use the singular form, no?